samples/subsys/usb/dfu/sample.yaml

```diff
@@ -29,6 +30,7 @@ common:
     - frdm_mcxa156
     - lpcxpresso55s69/lpc55s69/cpu0
     - stm32l562e_dk/stm32l562xx/ns
+    - stm32h7s78_dk
```

Comment:
Please remove test feature usb_device from https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/st/stm32h7s78_dk/stm32h7s78_dk.yaml

Reply:
But it seems, the stm32h7s78_dk does have a usb_device

Reply:
Yes, that is why you should remove the test feature usb_device instead of adding it to the exclude list.

Reply:
Because there are several boards in this list and it is getting bigger and bigger, and because there is **no test feature for the flash/flash-partitions/flash-foo-bar**, and because not every board needs `usb_device` test feature, and because I am the maintainer of this area and the noise caused by this changes really annoys me and always breaks someone else's patches just for nothing.

Reply:
OK removing from the `usb_device` entry of the stm32h7s78_dk.yaml